### PR TITLE
Add more checkpoint/restore information to 'inspect'

### PR DESCRIPTION
**Pull request by adrianreber**
This adds the following information to the output of 'podman inspect':

 * `CheckpointedAt` - time the container was checkpointed
   Only set if the container has been checkpointed

 * `RestoredAt` - time the container was restored
   Only set if the container has been restored

 * `CheckpointLog` - path to the checkpoint log file (CRIU's dump.log)
   Only set if the log file exists (--keep)

 * `RestoreLog` - path to the restore log file (CRIU's restore.log)
   Only set if the log file exists (--keep)

 * `CheckpointPath` - path to the actual (CRIU) checkpoint files
   Only set if the checkpoint files exists (--keep)

 * `Restored` - set to true if the container has been restored
   Only set if the container has been restored

### CODE REVIEW

**Comment by rhatdan** [comment]
:+1: 
/approve

File: libpod/define/container_inspect.go [hunk 1]
```diff
@@ -189,22 +189,28 @@ type InspectMount struct {
...
+	Pid            int                `json:"Pid"`
+	ConmonPid      int                `json:"ConmonPid,omitempty"`
+	ExitCode       int32              `json:"ExitCode"`
+	Error          string             `json:"Error"` // TODO
+	StartedAt      time.Time          `json:"StartedAt"`
+	FinishedAt     time.Time          `json:"FinishedAt"`
+	Health         HealthCheckResults `json:"Health,omitempty"`
+	Checkpointed   bool               `json:"Checkpointed,omitempty"`
+	CgroupPath     string             `json:"CgroupPath,omitempty"`
+	CheckpointedAt time.Time          `       json:"CheckpointedAt,omitempty"`
```
Comment (rhatdan) [inline libpod/define/container_inspect.go:208]: This looks like it could cause an issue.

Comment (adrianreber) [reply libpod/define/container_inspect.go]: I removed the unnecessary whitespace.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**